<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8" />
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=1, minimum-scale=1, maximum-scale=2.5" />
  <!-- Title, Description -->
  <title>Wiki</title>
  <meta name="description" content="Lorem ipsum dolor sit amet, consectetur adipisicing elit. Incidunt veritatis cupiditate ducimus modi culpa. Et ducimus excepturi molestiae alias.">
  <!-- Open Graph Metadata -->
  <meta property="og:url" content="" /> <!-- PAGE URL -->
  <meta property="og:type" content="website" /> <!-- TYPE -->
  <meta property="og:title" content="" /> <!-- TITLE -->
  <meta property="og:description" content="" /> <!-- DESCRIPTION-->
  <meta property="og:image" content="" /> <!-- IMAGE URL -->
  <!-- Stylesheets, Fonts, Neccessary Scripts -->
  <link rel="stylesheet" href="css/master.min.css">
  <!--PLOT.LY JavaScript-->
  <script src="js/plotly.js"></script>
</head>

<body>
  <div class="sticky-top">
    <nav class="navbar has-shadow" role="navigation" aria-label="main navigation" id="navbar">
      <div class="container">

        <div class="navbar-brand">
          <a class="navbar-logo" href="#">
            <img src="assets/T--UPF_CRG_Barcelona--logosensebarcelona.svg" alt="iGEM Barcelona team 2018">
          </a>
          <div class="navbar-burger" data-target="navbar-menu">
            <span></span>
            <span></span>
            <span></span>
          </div>
        </div>

        <!-- children of navbar-menu must be navbar-start and/or navbar-end -->
        <div class="navbar-menu" id="navbar-menu">
          <!-- navbar items -->
          <div class="navbar-grid">
            <!--class is-active for showing active menu-->
            <a class="navbar-item" href="Description.html">Project</a>
            <a class="navbar-item is-active" href="DryLab.html">DryLab</a>
            <a class="navbar-item" href="Parts.html">Parts</a>
            <a class="navbar-item" href="Deliverables.html">Deliverables</a>
            <a class="navbar-item" href="WetLab.html">WetLab</a>
            <a class="navbar-item" href="Intro_HP.html">Human Practices</a>
            <a class="navbar-item" href="Aboutteam.html">Team</a>
          </div>
        </div>
      </div>
    </nav>

    <header id="header">
      <div id="blur"></div>
      <!-- Swiper -->
      <div class="swiper-container" id="sw-container">
        <div class="swiper-wrapper">
          <a class="swiper-slide swiper-slide-menu" href="Validation.html">
            <embed rc="assets/T--UPF_CRG_Barcelona--exp_validation.svg" alt="Experimental Validation Doodle">
            <p>Experimental Validation</p>
          </a>
          <a class="swiper-slide swiper-slide-menu" href="Model.html">
            <embed rc="assets/T--UPF_CRG_Barcelona--model.svg" alt="Computationtal Model Doodle">
            <p>Model</p>
          </a>
          <a class="swiper-slide swiper-slide-menu" href="Design.html">
            <embed rc="assets/T--UPF_CRG_Barcelona--mage.svg" alt="Graphic Design Doodle">
            <p>Graphic Design</p>
          </a>
          <a class="swiper-slide swiper-slide-menu" href="Tutorials.html">
            <embed rc="assets/T--UPF_CRG_Barcelona--results.svg" alt="Tutorials doodle">
            <p>Tutorials</p>
          </a>
        </div>
        <!-- Add Arrows -->
        <div class="swiper-button-next"></div>
        <div class="swiper-button-prev"></div>
      </div>
    </header>
  </div>

  <main>

    <section class="container-centered">

      <div>
        <p class="page-title">MODEL</p>
        <div class="grid-threecolumns">
          <div>
            <a href="#Overview"><button class="button">Model overview</button></a>
          </div>
          <div>
            <a href="#fba"><button class="button">Flux Balance Analysis Models</button></a>
          </div>
          <div>
            <a href="#Simulation"><button class="button">Simulation of Intestine Conditions</button></a>
          </div>
          <div>
            <a href="#Switch"><button class="button">Metabolic Switch</button></a>
          </div>
        </div>

        <a name="Overview">
          <p class="subapart1">Model overview</p>
        </a>
        <p>It is very important to have in mind in which environment the genetically modified bacteria are going to be
          working. In the intestine, there is a steep gradient for oxygen and its concentrations decrease precipitously
         near anoxia in the middle of the lumen (Espey, 2013). For this reason, if we want our engineered
          bacteria to work in the gut, it has to work under anaerobic conditions. However, LCFA are degraded via the
          beta-oxidation pathway, an aerobic pathway for excellence, therefore it is needed to find alternative
          strategies in order to metabolize LCFA without oxygen. </p>
        <p>Moreover, when engineering a pathway in order to optimize it, it is very important to have in mind that the
          overall efficiency of the synthetic pathway can be reduced by the loss of metabolic intermediates by
          diffusion or by competent pathways (Na, Kim and Lee, 2010]. In the intestine, after diet ingest, there is
          going to be other energetic substrates aside of LCFA, especially sugars and amino acids. Therefore, most of
          the catabolic pathways of bacteria are going to be active and can interfere with beta oxidation efficiency.
          For this reason, it is very important to study the metabolic network as a whole instead of only focusing on
          the pathway that we want to modify. </p>

        <a name="fba">
          <p class=subapart1>Flux Balance Analysis Models</p>
        </a>
        <p>
          In order to simulate LCFA uptake and metabolism by <i>E. coli</i> we performed flux balance analysis (FBA),
          which is
          a mathematical approach for analyzing the flow of metabolites through a metabolic network (Orth, Thiele and
          Palsson, 2010) (see figure 1). FBA models make possible to predict cellular growth rate under different
          environmental conditions or genetic perturbations. Also, it allows to analyze the influx or production of a
          desired metabolite. For our models, we used the genome-scale metabolic network reconstruction of Escherichia
          coli K-12 MG1655 iJO1366 (Orth et al., 2014), which contain all the known metabolic reactions in our <i>E.
            coli</i>
          strain and the genes that encode each enzyme (Oberhardt, Palsson and Papin, 2009). The final goal was to find
          new strategies to engineer <i>E. coli</i> for achieving a bacterial system capable of efficiently uptaking
          LCFA in human gut-like conditions.
        </p>
        <img src="http://2018.igem.org/wiki/images/e/ec/T--UPF_CRG_Barcelona--FBA.png" alt="Figura FBA poster TFG" style="width: 70%">

        <a name="Simulation">
          <p class="subapart1">Simulation of Intestine Conditions</p>
        </a>
        <p class="subapart2">Aerobic vs anaerobic growth with LCFA</p>

        <p>First, we wanted to assess whether beta oxidation of LCFA was possible under anaerobic conditions. To do so,
          growth was predicted by optimizing the biomass function under either aerobic or anaerobic conditions. Each
          condition was analyzed with palmitate as the sole carbon source or glucose as a control variable. </p>

        <div id="graph_1" style="width:90%;height:400px;">
          <script>
            var wt_palmitate = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              y: [0, 0.229996059, 0.472515672, 0.715035284, 0.957554897, 1.20007451, 1.442594122, 1.685113735,
                1.927633348, 2.17015296, 2.412672573, 2.65519218560, 2.89771179827, 3.14023141094, 3.38275102361,
                3.62527063628, 3.86779024894, 4.11030986161, 4.35282947428, 4.59534908695
              ],
              name: 'Palmitic',
              type: 'scatter',
              marker: {
                color: "#0079bf",
              }
            };
            var wt_glucose = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              y: [0, 0.086478583, 0.186022275, 0.285565967, 0.385109659, 0.484653352, 0.584197044, 0.683740736,
                0.783284428, 0.88282812, 0.982371813, 1.081915505, 1.181459197, 1.281002889, 1.380546582,
                1.480090274, 1.579633966, 1.679177658, 1.778721351, 1.878265043,
              ],
              name: 'Glucose',
              type: 'scatter',
              marker: {
                color: "#c2ca6c",
              }
            };
            var anaerobic_palmitate = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              y: [0, 0, -0.052796042, -0.052796181, -0.05279604, -0.05279604, -0.05279604, -0.05279604, -0.05279604,
                -0.05279604, -0.05279604, -0.05279604, -0.05279604, -0.05279604, -0.05279604, -0.05279604, -
                0.05279604, -0.05279604, -0.05279604, -0.05279604
              ],
              name: 'Anaerobic palmitic',
              type: 'scatter',
              mode: 'markers',
              marker: {
                color: "#0079bf",
              }
            };
            var anaerobic_glucose = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              y: [0, 0, 0.020447524, 0.048079257, 0.075711014, 0.103342813, 0.130974529, 0.158606286, 0.186238043,
                0.2138698, 0.241501566, 0.269133314, 0.296765071, 0.324396829, 0.352028586, 0.379660341, 0.4072921,
                0.434923859, 0.462555618, 0.490187371,
              ],
              name: 'Anaerobic glucose',
              type: 'scatter',
              mode: 'markers',
              marker: {
                color: "#c2ca6c",
              }
            };
            var data = [wt_glucose, wt_palmitate, anaerobic_glucose, anaerobic_palmitate];
            var layout = {
              title: 'Anaerobic growth simulation with different electron acceptors',
              titlefont: {
                family: "PT Sans",
                size: 15,
                color: "#36393d",
              },
              xaxis: {
                title: "Maximum uptake rate",
                titlefont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
              },
              yaxis: {
                title: 'Growth rate',
                autorange: true,
                showticklabels: true,
                zeroline: false,
                autorange: true,
                titlefont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
                tickfont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
              },
            };
            Plotly.newPlot('graph_1', data, layout, {
              displayModeBar: false
            });
          </script>
        </div>
        <p class="fig-caption">Figure 1 | jdjjnf</p>

        <p>The model predicted no growth at all when PA was the only carbon source in anaerobic conditions (Figure 1).
          On the other hand, in aerobic conditions the simulations predicted a lot of growth (more than with only
          glucose*). <b>That means that beta-oxidation cannot be functional under anaerobic conditions </b>. When
          glucose was the only carbon source <i>E. coli</i> showed little growth compared to aerobic conditions as
          expected. </p>

        <p class=subapart2>Anaerobic growth with different electron acceptors</p>
        <p>The fact that in the simulated minimal medium bacteria could not grow with only palmitate as a carbon source
          under anaerobic conditions, can be explained by the lack of reductive power in the cell. This blocks the beta
          oxidation preventing the metabolism of palmitic acid. </p>
        <p>Alternative electron acceptors molecules have been shown to allow anaerobic respiration of sugars (Unden and
          Bongaerts, 1997). We hypothesized that these molecules could be able to oxidize LCFA too. Since the final
          objective is to make our bacteria work in a human gut-like environment, we were only interested in those
          electron acceptor molecules that could be found in the gut. Fumarate, nitrate and Trimethylamine N-oxide
          (TMAO) have been shown to allow anaerobic respiration and both can be found in the gut in normal conditions
          (Jones et al., 2011). Therefore, it was modelled the effect of these three molecules on the phenotype of
          <i>E. coli</i> under anaerobic conditions.</p>
        <p class=subapart2>Anaerobic growth with palmitate and glucose </p>

        <p>In the gut, PA is never found alone, normally there are also sugars that come from the diet. Therefore, it
          was analyzed how the uptake of PA is affected under different concentrations of glucose, the main sugar for
          excellence. To do that, a fixed concentration of palmitate was set and the biomass function was optimized for
          an increasing concentration of glucose. The nitrate available was set to be really low as we did before, in
          order to make it the limiting factor.</p>

        <div id="graph_2" style="width:90%;height:400px;">
          <script>
            var wt_palmitate = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              y: [0, 0, -0.052796042, -0.052796181, -0.05279604, -0.05279604, -0.05279604, -0.05279604, -0.05279604,
                -0.05279604, -0.05279604, -0.05279604, -0.05279604, -0.05279604, -0.05279604, -0.05279604, -
                0.05279604, -0.05279604, -0.05279604, -0.05279604
              ],
              name: 'Palmitic',
              type: 'scatter',
              marker: {
                color: "#0079bf",
              }
            };
            var wt_glucose = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              y: [0, 0, 0.020447523874698, 0.0480792573625964, 0.0757110144675361, 0.103342813469439,
                0.130974528677416, 0.158606285782357, 0.186238042887297, 0.213869799992238, 0.241501566051737,
                0.26913331420, 0.296765071307057, 0.324396828546013, 0.352028585516939, 0.379660341423004,
                0.407292099726817, 0.434923858847469, 0.462555617559701, 0.490187371041636
              ],
              name: 'Glucose',
              type: 'scatter',
              marker: {
                color: "#c2ca6c",
              }
            };
            var nitrate_palmitate = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              y: [0, 0.136442081384302, 0.289672934741247, 0.442903788098192, 0.596134636907663, 0.749365490264608,
                0.902596343442927, 1.055827197, 1.20905805, 1.362288904, 1.515519757, 1.66875061, 1.821981464,
                1.975212317, 2.128443171, 2.281674024, 2.434904882, 2.588135731, 2.741366583, 2.894597433
              ],
              name: 'Palmitic_nitrate',
              type: 'scatter',
              mode: 'markers',
              marker: {
                color: "#0079bf",
              }
            };

            var nitrate_glucose = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              y: [0, 0.075027774, 0.165970532, 0.256913286, 0.347856044, 0.438798798, 0.52974155, 0.620684313,
                0.71162707, 0.802569832, 0.893512579, 0.984455339, 1.075398095, 1.166340858, 1.257283609,
                1.34822636, 1.439169118, 1.530111878, 1.621054632, 1.711997388
              ],
              name: 'Glucose_nitrate',
              type: 'scatter',
              mode: 'markers',
              marker: {
                color: "#c2ca6c",
              }
            };
            var tmao_palmitic = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              y: [0, -0.0435845059232994, -0.0435845059232996, -0.0435845059232997, -0.0435845059232994, -
                0.0435845059232994, -0.0435845059232993, -0.0435845059232994, -0.0435845059232995, -
                0.0435845059232996, -0.0435845059232994, -0.0435845059232996, -0.0435845059232996, -
                0.0435845059232993, -0.0435845059232996, -0.0435845059232996, -0.0435845059232995, -
                0.0435845059232998, -0.0435845059232997, -0.0435845059232994
              ],
              name: 'Palmitic_TMAO',
              type: 'scatter',
              mode: 'markers',
              marker: {
                color: "#0079bf",
                symbol: "square",
              }
            };
            var tmao_glucose = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              y: [0, 0.0424658429089024, 0.111160588790953, 0.179855334672997, 0.248550080555252, 0.317244826437098,
                0.385939572319142, 0.454634318201201, 0.523329064083245, 0.592023855818731, 0.660718555847334,
                0.729413301729391, 0.798108047611444, 0.866802793493473, 0.935497539375541, 1.00419228525758,
                1.07288703113964, 1.14158177702167, 1.21027652290374, 1.2789712687858
              ],
              name: 'Glucose_TMAO',
              type: 'scatter',
              mode: 'markers',
              marker: {
                color: "#c2ca6c",
                symbol: "square",
              }
            };
            var fumarate_palmitic = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              y: [0, 0.0729514559456984, 0.0729514559456983, 0.0729514559456978, 0.0729514559456978,
                0.0729514559456978, 0.0729514559456978, 0.0729514559456988, 0.0729514559456984, 0.072951455946407,
                0.0729514559457008, 0.0729514559456989, 0.0729514559452601, 0.0729514559456976, 0.0729514559456968,
                0.0729514559457012, 0.0729514559457171, 0.0729514559470517, 0.072951455945701, 0.072951455945167
              ],
              name: 'Palmitic_fumarate',
              type: 'scatter',
              mode: 'markers',
              marker: {
                color: "#0079bf",
                symbol: "star-diamond",
              }
            };
            var fumarate_glucose = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              y: [0, 0, 0.020447523874698, 0.0480792573625964, 0.0757110144675361, 0.103342813469439,
                0.130974528677416, 0.158606285782357, 0.186238042887297, 0.213869799992238, 0.241501566051737,
                0.269133314202118, 0.296765071307057, 0.324396828546013, 0.352028585516939, 0.379660341423004,
                0.407292099726817, 0.434923858847469, 0.462555617559701, 0.490187371041636
              ],
              name: 'Glucose_fumarate',
              type: 'scatter',
              mode: 'markers',
              marker: {
                color: "#c2ca6c",
                symbol: "star-diamond",
              }
            };
            var data = [wt_glucose, wt_palmitate, nitrate_glucose, nitrate_palmitate, fumarate_glucose,
              fumarate_palmitic, tmao_glucose, tmao_palmitic
            ];
            var layout = {
              title: 'Anaerobic growth simulation with different electron acceptors',
              titlefont: {
                family: "PT Sans",
                size: 15,
                color: "#36393d",
              },
              xaxis: {
                title: "Maximum uptake rate",
                titlefont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
              },
              yaxis: {
                title: 'Growth rate',
                autorange: true,
                showticklabels: true,
                zeroline: false,
                autorange: true,
                titlefont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
                tickfont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
              },
            };
            Plotly.newPlot('graph_2', data, layout, {
              displayModeBar: false
            });
          </script>
        </div>

        <p class="fig-caption">Figure 2 | jdjjnf</p>

        <p>Growth rate was predicted under anaerobic conditions with an arbitrary high concentration of nitrate,
          fumarate or TMAO in the simulated medium. When nitrate was present, both palmitate and glucose were able to
          be oxidized (Figure 2), indicating that nitrate was the final electron acceptor in the phosphorylative
          oxidation. On the other hand, neither fumarate nor TMAO were able to oxidize PA. </p>
        <p>Gut concentrations of nitrate and fumarate are not well known due to the fact that is very difficult to
          quantify molecule concentrations in vivo in the intestine. However, in healthy conditions their
          concentrations are not very high (Jones et al., 2007). For this reason, in order to model PA uptake, we set a
          low concentration of nitrate in the medium for all the models (20 mmol gDW<sup>-1 </sup>hr<sup>-1</sup> ,
          millimoles per gram dry cell weight per hour). </p>

        <p class=subapart2>Graph 3</p>
        <div id="graph_3" style="width:90%;height:400px;">
          <script>
            var uptake_glucose = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26,
                27, 28, 29
              ],
              y: [10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10,
                10, 10, 10, 10, 10,
              ],
              name: 'Glucose Uptake',
              type: 'scatter',
              marker: {
                color: "#c2ca6c",
              }
            };
            var uptake_palmitate = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26,
                27, 28, 29
              ],
              y: [0.999814085, 1.093689037, 1.18756399, 1.281438942, 1.375313894, 1.469188847, 1.563063799,
                1.656938751, 1.750813704, 1.844688656, 1.938563609, 2.032438561, 2.126313513, 2.220188466,
                2.314063418, 2.40793837, 2.501813323, 2.595688275, 2.689563227, 2.78343818, 2.877313132,
                2.971188084, 3.065063037, 3.158937989, 3.252812942, 3.346687894, 3.440562846, 3.534437799,
                3.628312751, 3.722187703
              ],
              name: 'Palmitate Uptake',
              type: 'scatter',
              marker: {
                color: "#577d95",
              }
            };

            var growth = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26,
                27, 28, 29
              ],
              y: [0.938295419, 0.910513661, 0.882731903, 0.854950145, 0.827168387, 0.799386629, 0.771604871,
                0.743823113, 0.716041355, 0.688259597, 0.660477839, 0.632696081, 0.604914323, 0.577132565,
                0.549350807, 0.52156905, 0.493787292, 0.466005534, 0.438223776, 0.410442018, 0.38266026,
                0.354878502, 0.327096744, 0.299314986, 0.271533228, 0.24375147, 0.215969712, 0.188187954,
                0.160406196, 0.132624438
              ],
              name: 'Growth Rate',
              yaxis: 'y2',
              type: 'scatter',
              marker: {
                color: "#412350",
              }
            };

            var data = [uptake_glucose, uptake_palmitate, growth];

            var layout = {
              title: 'No té titol',
              titlefont: {
                family: "PT Sans",
                size: 15,
                color: "#36393d",
              },
              xaxis: {
                title: "Lactate production",
                titlefont: {
                  family: "PT Sans",
                  size: 13,
                  color: "#36393d",
                },
              },
              yaxis: {
                title: 'Uptake rate',
                autorange: true,
                showticklabels: true,
                zeroline: false,
                titlefont: {
                  family: "PT Sans",
                  size: 13,
                  color: "#36393d",
                },
                tickfont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
              },
              yaxis2: {
                title: 'Growth rate',
                autorange: true,
                showticklabels: true,
                zeroline: false,
                autorange: true,
                titlefont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
                tickfont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
                overlaying: 'y',
                side: 'right',
              },
            };
            Plotly.newPlot('graph_3', data, layout, {
              displayModeBar: false
            });
          </script>
        </div>
        <p class="fig-caption">Figure 3 | jdjjnf</p>

        <p>The model predicted a reduction of PA uptake (from 2.5 to 0.4 mmol gDW<sup>-1 </sup> hr<sup>-1 </sup>) when
          glucose concentration was increased (Figure 3). Nevertheless, when glucose concentration overcame a certain
          threshold, approximately at 14 mmol, palmitate uptake got stabilized at 0.4 mmol (Figure 3). These results
          indicate that bacteria, in order to maximize their growth, when both PA and glucose are present and there is
          little reductive power, prioritize the oxidation of glucose before palmitate. When the reductive power is not
          enough to fully oxidize all the glucose available, glucose is partially fermented. When this happens,
          palmitate uptake does not decrease anymore for any increase in glucose concentration.</p>


        <p class=subapart2>Finding strategies to priorize LCFA oxidation over sugars oxidation</p>
        <p>After the previous results, we hypothesized that an increase of glucose fermentation would increase PA
          uptake. In order to check the hypothesis, it was optimized the production of lactate using FBA in order to
          force the organism to ferment glucose. To do so, the lower bound of the lactate exchange reaction was set to
          be different positive values.</p>


        <div id="graph_4" style="width:90%;height:400px;">
          <script>
            var uptake_glucose = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              y: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              name: 'Glucose Uptake',
              type: 'scatter',
              marker: {
                color: "#0079bf",
              }
            };
            var uptake_palmitate = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              y: [2.495245375, 2.363192096, 2.213396909, 2.061699056, 1.910001203, 1.75830335, 1.606605497,
                1.454907644, 1.303209791, 1.151511938, 0.999814085, 0.848116232, 0.696418379, 0.539605421,
                0.382675316, 0.396134278, 0.410469656, 0.424805034, 0.439140412, 0.45347579
              ],
              name: 'Palmitate Uptake',
              type: 'scatter',
              marker: {
                color: "#c2ca6c",
              }
            };

            var growth = {
              x: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19],
              y: [0.205666742, 0.287031811, 0.360177912, 0.432442601, 0.504707289, 0.576971977, 0.649236666,
                0.721501354, 0.793766042, 0.86603073, 0.938295419, 1.010560107, 1.082824795, 1.152719903,
                1.222560742, 1.268563352, 1.314470349, 1.360377346, 1.406284343, 1.45219134
              ],
              name: 'Growth Rate',
              yaxis: 'y2',
              type: 'scatter',
              marker: {
                color: "#412350",
              }
            };

            var data = [uptake_glucose, uptake_palmitate, growth];

            var layout = {
              title: 'No té titol',
              titlefont: {
                family: "PT Sans",
                size: 15,
                color: "#36393d",
              },
              xaxis: {
                title: "Glucose Maximum uptake rate",
                titlefont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
              },
              yaxis: {
                title: 'Uptake rate',
                autorange: true,
                showticklabels: true,
                zeroline: false,
                autorange: true,
                titlefont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
                tickfont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
              },
              yaxis2: {
                title: 'Growth rate',
                autorange: true,
                showticklabels: true,
                zeroline: false,
                autorange: true,
                titlefont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
                tickfont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
                overlaying: 'y',
                side: 'right',
              },
            };
            Plotly.newPlot('graph_4', data, layout, {
              displayModeBar: false
            });
          </script>
        </div>
        <p>Our models predicted that an increase of lactate production allowed an increase of palmitate uptake (Figure
          4). PA uptake flux went from 0.99 to 2.81 mmol gDW<sup>-1 </sup> hr<sup>-1 </sup>, when there was no lactate
          production and when there was 19 mmol gDW<sup>-1 </sup> hr<sup>-1 </sup> of lactate production respectively
          (Figure 4). Biomass was reduced from 0.93 to 0.41 mmol gDW<sup>-1 </sup> hr<sup>-1 </sup>.</p>


        <p class=subapart2>Creating the perfect intestine PA eater</p>
        <p>To achieve this behavior for our bacteria, we would have to genetically modify it. One simplistic option
          would be to overexpress the lactate dehydrogenase gene. However, this system would not be very effective
          because of blab bla and we would be creating a metabolic burden that would reduce bacterial growth a lot. For
          this reason we decided to find genome knock outs that would cause the same effect. </p>

        <p>We implemented an algorithm to compare the metabolic flow between the wildtype strain and a simulated lactate
          producer strain, with that, we were able to find possible targets as knock outs. We designed another
          algorithm to simulate the effects of the target knock outs on the biomass and PA uptake and we iterated both
          algorithms until achieving a substantial increase in PA uptake without creating a letal genome modification.
          We ended up with a list of 63 knock outs that together produced the desirable effect. However, it would be
          unfeasible to perform that many knock outs. We designed one more algorithm to refine the large list and rule
          out the redundant gene knock outs. We achieved to find a combination of 6 knock out that were able to
          increase a lot PA uptake.</p>

     <div id="palmitate_uptake" style="width:90%;height:400px;">
          <script>
            var palmitate_up = {
              x: ['Wildtype', '6 KO', '7 KO', '5 KO', '6 KO only glucose', '6 KO only palmitate'],
              y: [0.45347579, 1.788431437, 2.04101627, 0.446545532, 0, 2.495197114],
              name: 'Palmitate Uptake',
              type: 'bar',
              marker: {
                color: '#0079bf',
              }
            };
            var growth = {
              x: ['Wildtype', '6 KO', '7 KO', '5 KO', '6 KO only glucose', '6 KO only palmitate'],
              y: [1.45219134, 0.928854207, 0.841675175, 1.42999818, 0.623268257, 0.205644384],
              name: 'Growth',
              type: 'bar',
              marker: {
                color: "#c2ca6c",
              }
            };
            var data = [palmitate_up, growth];
            var layout = {
              barmode: 'group',
              title: "Palmitate Uptake with glucose present",
              titlefont: {
                family: "PT Sans",
                size: 15,
                color: "#36393d",
              },
              xaxis: {
                autorange: true,
                showticklabels: true,
                tickfont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
              },
              yaxis: {
                zeroline: false,
                autorange: true,
                title: "Growth rate / palmitate uptake",
                titlefont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
                showticklabels: true,
                tickfont: {
                  family: "PT Sans",
                  size: 11,
                  color: "#36393d",
                },
              },
            };
            Plotly.newPlot('palmitate_uptake', data, layout, {
              displayModeBar: false
            });
          </script>
        </div>

        <a name="Switch">
          <p class="subapart1">Metabolic Switch</p>
        </a>
      </div>

    </section>

  </main>

  <!-- Footer -->
  <footer class="footer">
    <div>
      <p class="subapart2">Follow us at:</p>
      <a href="https://www.instagram.com/igembarcelona/"><embed class="social-svg" src="assets/T--UPF_CRG_Barcelona--instagram.svg"
          alt=""></a>
      <a href="https://twitter.com/igemBarcelona"><embed class="social-svg" src="assets/T--UPF_CRG_Barcelona--twitter.svg"
          alt=""></a>
      <a href="https://www.facebook.com/igembarcelona/""><embed class=" social-svg" src="assets/T--UPF_CRG_Barcelona--facebook.svg"
        alt=""></a>
    </div>

    <div>
      <center>
        <p class="subapart2">For Judges</p>
      </center>
      <div id="judges">
        <a href="Attributions.html">Attributions</a>
        <a href="Collaborations.html">Collaborations</a>
        <a href="Demonstrate.html">Demonstration</a>
        <a href="Human_Practices.html">Human Practices</a>
        <a href="InterLab.html">InterLab</a>
        <a href="Improve.html">Improved part</a>
        <a href="http://igem.org/2018_Judging_Form?id=2581">Judging Form</a>
        <a href="Model.html">Model</a>
        <a href="Parts">Parts</a>
      </div>
    </div>

    <div class="footer-contact">
      <p class="subapart2">Contact us:</p>
      <p>igem [at] upf [dot] edu</p>
      <p> PRBB Building</p>
      <p>Doctor Aiguader 88, 08003 Barcelona (Spain)</p>
    </div>

    </div>
  </footer>

  <!-- JS code -->
  <script src="js/scripts.js"></script>
  <script src="js/devel.js"></script>
  <script>
    // Initialize Swiper
    var swiper = new Swiper('.swiper-container', {
      slidesPerView: 2,
      slidesPerGroup: 1,
      loop: true,
      freeMode: true,
      observer: true,
      watchOverflow: true,
      navigation: {
        nextEl: '.swiper-button-next',
        prevEl: '.swiper-button-prev',
      },
      keyboard: {
        enabled: true,
      },
      watchOverflow: true,
      freeModeSticky: true,
      mousewheel: {
        forceToAxis: true,
      },
      preventClicks: true,
      iOSEdgeSwipeDetection: true,
      observeParents: true,
      breakpoints: {
        1024: {
          slidesPerView: 5,
        },
        768: {
          slidesPerView: 4,
        },
        640: {
          slidesPerView: 3,
        },
        500: {
          slidesPerView: 2,
        },
        320: {
          slidesPerView: 1,
        }
      },
    });
  </script>
</body>

</html>